xarray/backends/plugins.py

```diff
@@ -112,9 +112,14 @@ def guess_engine(store_spec):
 
 def get_backend(engine):
     """Select open_dataset method based on current engine"""
-    engines = list_engines()
-    if engine not in engines:
-        raise ValueError(
-            f"unrecognized engine {engine} must be one of: {list(engines)}"
-        )
-    return engines[engine]
+    if hasattr(engine, "open_dataset"):
```

Comment:
Sorry, 
I added this comment one month ago, but I didn't know that I had to the button submit and remove the tag "pending"
I'm so stupid!

Reply:
I think requiring subclassing is bit restrictive. I'm of the opinion that if the user insists on using this then it's the users responsibility to make sure it works all the way.  

Since `open_dataset` is the main event of the `BackendEntrypoint` I thought that's enough of a check to just assume this engine works as expected.

Reply:
I yield to the majority. Let's require to have subclasses of xarray,backends.BackendEntrypoint.